rest/handler.go

```diff
@@ -164,6 +164,10 @@ func newHandler(server *ServerContext, privs handlerPrivs, r http.ResponseWriter
 		startTime:    time.Now(),
 		runOffline:   runOffline,
 	}
+
+	h.rqCtx = base.LogContextWith(h.rq.Context(), &base.LogContext{CorrelationID: h.formatSerialNumber()})
```

Comment:
Do we need to set this here if we call h.ctx() consistently elsewhere?

Reply:
sure, will do.